ev3dev2/wheel.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+"""
+Wheel and Rim classes
+
+A great reference when adding new wheels is http://wheels.sariel.pl/
+"""
+from math import pi
+
+
+class Wheel(object):
+    """
+    A base class for various types of wheels, tires, etc.  All units are in mm.
+
+    One scenario where one of the child classes below would be used is when the
+    user needs their robot to drive at a specific speed or drive for a specific
+    distance. Both of those calculations require the circumference of the wheel
+    of the robot.
+    """
+
+    def __init__(self, diameter_mm, width_mm):
+        self.diameter_mm = float(diameter_mm)
+        self.width_mm = float(width_mm)
+        self.circumference_mm = diameter_mm * pi
+
+    @property
+    def radius_mm(self):
```

Comment:
Can you add a docs page so that these classes show up? I think we can follow the model from the unit section of the motor page (https://python-ev3dev.readthedocs.io/en/ev3dev-stretch/motors.html#units).

Reply:
Is there a howto on writing docs? It has been ages since I have tried to edit them. What repo are they in?

Reply:
There are files in this repo's `docs/` which declare the top-level structure. Many of the sections just have a tag to auto-gen docs from doc comments of a class. You can preview by either manually running Sphinx locally or enabling the branch as a private version in RTD.

Reply:
I checked something in but I am not familiar with Sphinx or RTD (Real Time Documentation?) so I do not know how to check the end result.  Unfortunately googling for "ev3dev sphinx" gets a hit on every page of our docs due to the "Built with Sphinx using..." text at the bottom.

Reply:
ok I initially created a wheels.rst but then the build failed because that was never referenced from anywhere (doh!).  I started looking at where it should be referenced and then realized that all of the these wheel docs are probably a better fit in others.rst instead of being a standalone rst file.